client/src/components/SideBarHeader.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/core/styles";
+
+import MoreHorizIcon from "@material-ui/icons/MoreHoriz";
+import happyChatter from "../assets/5e4a118a03239f87632c8f33899048fd2c4af1ca.png";
```

Comment:
Can we give the files corresponding names?
(`import happyChatter from "../assets/happyChatter.png";`)